### PR TITLE
[SPARK-8456][ML]Ngram featurizer python

### PR DESCRIPTION
**Pull request by feynmanliang**
Python API for N-gram feature transformer

### CODE REVIEW

File: python/pyspark/ml/feature.py [hunk 1]
```diff
@@ -266,6 +266,76 @@ class IDFModel(JavaModel):
 
 
 @inherit_doc
+@ignore_unicode_prefix
+class NGram(JavaTransformer, HasInputCol, HasOutputCol):
+    """
+    A feature transformer that converts the input array of stringsinto an array of n-grams. Null
```
Comment (jkbradley) [inline python/pyspark/ml/feature.py]: typo

**Comment by jkbradley** [comment]
Looks good save for the typo + the remaining update for the doc test

File: python/pyspark/ml/feature.py (hunk 1 above)
Comment (feynmanliang) [inline python/pyspark/ml/feature.py]: OK.

**Comment by jkbradley** [comment]
LGTM merging into master
Thanks!

**Pull request closed without merging**